### DISCUSSION TITLE
Looking for examples of custom CSS

### DISCUSSION THREAD

**Discussion by fccfc5d245c** [Athou/commafeed#1570] [General]
I don't know CSS but I can tweak existing code, so I'm searching for user-created CSS themes and have found very little (see the attached code, I found it on Issues and I used it to change some fonts and add a colour, as a test)

I've seen the recommended [tutorial](https://www.w3schools.com/Css/), but learning all of this, only to tweak the appearance of something that already looks good... while it would be cool to learn, seems like investing too much time for me: I'll never work as a dev, I'm on art and graphic design.

So, if we don't have a collection of themes posted anywhere, I suggest creating a thread here where we can share our aesthetic experiments and learn / get inspired from each other.

If that idea doesn't fly, I'd humbly request a basic CSS with a generous amount of tweakable elements (changing the whole color palette would be ideal) already written, so software ignorants like myself can play with it and maybe make something valuable for other users.

```
margin-right:10px;
}
.css-treeview a, .css-treeview {
font-size: 14px;
line-height:24px;
font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}
.spinner {
padding:0 16px 0 0;
}
# feed-accordion .entry-buttons {
margin-top: 10px;
}
body {
color: rgb(0, 204, 255);
font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
font-size: 20px;
}
# feed-accordion .entry-body-content {

max-width: 1360px;
}
.entry-body img {
margin-bottom: 15px !important;
margin: 0 auto;
}```
```

**Comment by ZipMartini** [comment]
One thing you can do is use the Elements tab of the Inspect tool of your browser to investigate an element on the page (usually Crtl-Shit-I).  That element should have a class parameter inside of its div tag... probably more than one.  Find the one that's unique to that kind of element, and you can alter its CSS.

Here's my example.  I've made it so that every category in the tree on the left is a black bar with bold white text, and the feeds under the categories all have orange text:

```
.cf-treenode-category{
    font-weight:bold;
    color:#ffffff;
    background-color: #000000;
}
.cf-treenode-feed{
    font-weight:normal;
    color:#FD954E;
}
```

**Reply by fccfc5d245c** [reply]
That'll be useful. Thank you very much!
